INSTALL.md

```diff
@@ -3,7 +3,7 @@
 ## Windows (Partially tested)
 
  ### 1. Prepare prerequisites
- * Install [Python](https://www.python.org).
+ * Install [Python](https://www.python.org) 3.7. Any other version will not work.
```

Comment:
My suggestion is to reset this commit with the diff containing "Any other version **may not** work", assuming you have not tested other than 3.7 and you know you need the python runtime to be between 3.6 and 3.7.x

Reply:
Ok, I'll do that.